gapic-generator/lib/gapic/presenters/method_presenter.rb

```diff
@@ -232,6 +232,19 @@ def grpc_method_name
         @method.name
       end
 
+      ##
+      # Returns a hash with a drift_manifest of this rpc method
+      # describing correspondence between the proto description
+      # of the rpc with the generated code for the method.
+      # For ruby currently [03/2021] only one method is generated per RPC,
+      # so the correspondence is very basic.
+      # See https://github.com/googleapis/googleapis/blob/master/gapic/metadata/gapic_metadata.proto
+      #
+      # @return [Hash]
+      def drift_manifest
```

Comment:
Is it possible for you to populate the `gapic_metadata.proto` and then call a standard proto-to-JSON serializer on the proto? This would avoid discrepancies like the one you're fixing, and would be useful should we add new features in the future.

Reply:
it's not possible to use the ruby classes generated by the standard proto-to-JSON serializer inside the ruby microgenerator at the moment

Reply:
So there is no standard code to take a proto populated by Ruby and emit JSON? Bummer.